tests/phpunit/src/Commands/Email/ConfigurePlatformEmailCommandTest.php

```diff
@@ -581,4 +686,16 @@ public function testConfigurePlatformEmailWithAlreadyEnabledEnvs($base_domain, $
 
   }
 
+  /**
+   * @param \Prophecy\Prophecy\ObjectProphecy $local_machine_helper
+   *
+   * @return \Prophecy\Prophecy\ObjectProphecy
+   */
+  protected function mockGetFilesystem(ObjectProphecy $local_machine_helper) {
```

Comment:
@danepowell I added this because the TestBase implementation of this method returns an actual FileSystem and not a Prophecy